klippy/kinematics/extruder.py

```diff
@@ -205,8 +209,16 @@ def move(self, print_time, move):
         self.extruder_move_fill(
             self.cmove, print_time, accel_t, cruise_t, decel_t, start_pos,
             start_v, cruise_v, accel, extra_accel_v, extra_decel_v)
-        self.stepper.step_itersolve(self.cmove)
+        for s in self.stepper_list:
+            s.step_itersolve(self.cmove)
         self.extrude_pos = start_pos + axis_d
+    def add_stepper(self, stepper):
+        if stepper == None:
+            return
```

Comment:
I'd avoid using an equals comparison with None - probably best to just omit this check.

Reply:
Now when I really think about it, apparently I wasn't thinking at all when writing that. Will remove the check.